config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows-presubmits.yaml

```diff
@@ -0,0 +1,217 @@
+# generated by ./config/jobs/kubernetes-sigs/sig-windows/generate-presubmits.sh.
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-aks-engine-windows-containerd-1-25
```

Comment:
We should remove this entry (and any other entry for aks-engine)

Reply:
addressed via commit : 3902356dd84622b0000f1b7c4ac5893017824c4e